File: JAVA/2160. Minimum Sum of Four Digit Number After Splitting Digits.java
class Solution {
    public int minimumSum(int n) {
        int[] a=new int[4];
        for(int i=0;i<4;i++){
            a[i]=n%10;
            n/=10;
        }
        Arrays.sort(a);
        return((10*a[0]+a[2])+(10*a[1]+a[3]));
    }
}

File: C++/2160. Minimum Sum of Four Digit Number After Splitting Digits.cpp
class Solution {
public:
    int minimumSum(int n) {
        vector<int>a;
        while(n!=0){
            a.push_back(n%10);
            n/=10;
        }
        sort(a.begin(),a.end());
        return(((10*a[0])+a[2])+((10*a[1])+a[3]));
    }
};